tutorial/lesson_07_multi_stage_pipelines.cpp

```diff
@@ -30,7 +31,7 @@ int main(int argc, char **argv) {
     // first horizontally, and then vertically.
     {
         // Take a color 8-bit input
-        Image<uint8_t> input = load<uint8_t>("images/rgb.png");
+        Image<uint8_t> input = load<Image<uint8_t>>("images/rgb.png");
```

Comment:
I don't want the audience to think about what I'm writing when I write that line (or when they read that line in the tutorials). I want them to mentally dismiss it as an obvious way to load an image and not even consider error conditions. How about:

bool load_image(string file, ImageType* im); // false == failure
ImageType load_image(string file);  // aborts or throws exception